jekyll/_cci2/server/air-gapped-installation/phase-3-installing-circleci-server.adoc

```diff
@@ -0,0 +1,108 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Phase 3 - Installing CircleCI Server
+:page-layout: classic-docs
+:page-liquid:
+:page-description: How to install the CircleCI Server Helm deployment to an air-gapped environment.
+:icons: font
+:toc: macro
+:toc-title:
+
+With prerequisites installed and object storage configured, we can now copy over and install the CircleCI helm deployment to the Kubernetes cluster in the air-gapped environment.
+
+[#preparing-values-yaml]
+== Preparing Values.yaml
+The values.yaml file for installing CircleCI should be prepared according to the https://circleci.com/docs/server/installation/phase-2-core-services/[regular installation instructions], after which, modifying the fields listed in the steps below for air-gap compatibility.
+
+=== 1. Configure Min.IO Build Agent
+In the distributor section of the values.yaml file, point `agent_base_url` and `launch_agent_base_url` to your Min.IO installation, and the created circleci-data bucket from phase 2.
+
+[source, yaml]
+----
+distributor:
+  agent_base_url: http://<minio-internal-tld>:9000/circle-data/
+  launch_agent_base_url: http://<minio-internal-tld>:9000/circle-data/
+----
+
+[#configure-minio-storage]
+=== 2. Configure Min.IO Object Storage
+In the object_storage section of the values.yaml, add the following configuration, modifying ports as necessary.
+
+[source, yaml]
+----
+object_storage:
+  bucketName: circle-data
+  expireAfter: 0
+  s3:
+    accessKey: <minio-root-user>
+    enabled: true
+    endpoint: http://<minio-internal-tld>:9000
+    secretKey: <minio-root-password>
+----
+
+[#configure-build-agent-image]
+=== 3. Configure the Nomad build agent image
+Specify the location of the Nomad build agent image within your registry, copied during phase 1, modifying the port as necessary.
+
+[source, yaml]
+----
+nomad:
+  buildAgentImage: "<your-internal-registry-tld>:5000/circleci/picard"
+----
+
+[#configure-docker-registry]
+=== 4. Configure your internal docker registry
```

Comment:
This is covered in the regular docs: https://circleci.com/docs/server/installation/phase-2-core-services/#Registry